### ISSUE TITLE
New update lua error... 1/4/2023

### ISSUE THREAD

**Issue by Milky063** [AdiAddons/AdiBags#867]
### Addon Version

AdiBags-v1.9.51.zip

### World of Warcraft Version

Retail

### Describe the bug

as i was logging into the game the error happen in the loading screen. 

1x AdiBags/core/Core.lua:122: attempt to call field 'SetCVarBool' (a nil value)
[string "@AdiBags/core/Core.lua"]:122: in function <AdiBags/core/Core.lua:82>
[string "=[C]"]: ?
[string "=[C]"]: ?
[string "@AdiBags/libs/AceAddon-3.0-13/AceAddon-3.0.lua"]:66: in function <...ceAdiBags/libs/AceAddon-3.0/AceAddon-3.0.lua:61>
[string "@AdiBags/libs/AceAddon-3.0-13/AceAddon-3.0.lua"]:494: in function `InitializeAddon'
[string "@AdiBags/libs/AceAddon-3.0-13/AceAddon-3.0.lua"]:619: in function <...ceAdiBags/libs/AceAddon-3.0/AceAddon-3.0.lua:611>

### Steps to reproduce

closed the game and started the game back up. 

### Verification

- [X] I have disabled all other addons and made sure this bug is triggered only with AdiBags enabled

**Comment by Frurox** [comment]
Getting this too:
```
Message: Interface/AddOns/AdiBags/core/Hooks.lua:216: attempt to call field 'CloseSpecialWindows' (a nil value)
Time: Wed Jan  4 22:56:45 2023
Count: 2
Stack: Interface/AddOns/AdiBags/core/Hooks.lua:216: attempt to call field 'CloseSpecialWindows' (a nil value)
[string "=[C]"]: in function `CloseSpecialWindows'
[string "@Interface/AddOns/AdiBags/core/Hooks.lua"]:216: in function <Interface/AddOns/AdiBags/core/Hooks.lua:215>
[string "=(tail call)"]: ?
[string "=[C]"]: ?
[string "=[C]"]: in function `securecall'
[string "@Interface/FrameXML/UIParent.lua"]:3393: in function `CloseWindows'
[string "@Interface/FrameXML/UIParent.lua"]:3410: in function <Interface/FrameXML/UIParent.lua:3408>
[string "=[C]"]: in function `securecall'
[string "@Interface/FrameXML/UIParent.lua"]:4059: in function `ToggleGameMenu'
[string "TOGGLEGAMEMENU"]:1: in function <[string "TOGGLEGAMEMENU"]:1>

Locals: 
```
and...
```
Message: Interface/AddOns/AdiBags/core/Hooks.lua:216: attempt to call field 'CloseSpecialWindows' (a nil value)
Time: Wed Jan  4 22:56:19 2023
Count: 1
Stack: Interface/AddOns/AdiBags/core/Hooks.lua:216: attempt to call field 'CloseSpecialWindows' (a nil value)
[string "=[C]"]: in function `CloseSpecialWindows'
[string "@Interface/AddOns/AdiBags/core/Hooks.lua"]:216: in function <Interface/AddOns/AdiBags/core/Hooks.lua:215>
[string "=(tail call)"]: ?
[string "=[C]"]: ?
[string "=[C]"]: in function `securecall'
[string "@Interface/FrameXML/UIParent.lua"]:3393: in function <Interface/FrameXML/UIParent.lua:3362>
[string "=[C]"]: in function `securecall'
[string "@Interface/FrameXML/UIParent.lua"]:2689: in function `ShowUIPanel'
[string "@Interface/FrameXML/UIParent.lua"]:2598: in function <Interface/FrameXML/UIParent.lua:2594>
[string "=[C]"]: in function `SetAttribute'
[string "@Interface/FrameXML/UIParent.lua"]:3196: in function `ShowUIPanel'
[string "@Interface/FrameXML/UIParent.lua"]:4078: in function `ToggleGameMenu'
[string "TOGGLEGAMEMENU"]:1: in function <[string "TOGGLEGAMEMENU"]:1>

Locals: 
```

**Comment by asymptotic1** [comment]
Same.  Easiest way to trigger it to hit escape.

Message: Interface/AddOns/AdiBags/core/Core.lua:122: attempt to call field 'SetCVarBool' (a nil value)
Time: Wed Jan  4 20:58:30 2023
Count: 1
Stack: Interface/AddOns/AdiBags/core/Core.lua:122: attempt to call field 'SetCVarBool' (a nil value)
[string "@Interface/AddOns/AdiBags/core/Core.lua"]:122: in function <Interface/AddOns/AdiBags/core/Core.lua:82>
[string "=[C]"]: ?
[string "=[C]"]: ?
[string "@Interface/AddOns/AdiBags/libs/AceAddon-3.0/AceAddon-3.0.lua"]:66: in function <...ce/AddOns/AdiBags/libs/AceAddon-3.0/AceAddon-3.0.lua:61>
[string "@Interface/AddOns/AdiBags/libs/AceAddon-3.0/AceAddon-3.0.lua"]:494: in function `InitializeAddon'
[string "@Interface/AddOns/AdiBags/libs/AceAddon-3.0/AceAddon-3.0.lua"]:619: in function <...ce/AddOns/AdiBags/libs/AceAddon-3.0/AceAddon-3.0.lua:611>

Locals: <none>

Message: Interface/AddOns/AdiBags/core/Core.lua:122: attempt to call field 'SetCVarBool' (a nil value)
Time: Wed Jan  4 20:58:30 2023
Count: 1
Stack: Interface/AddOns/AdiBags/core/Core.lua:122: attempt to call field 'SetCVarBool' (a nil value)
[string "@Interface/AddOns/AdiBags/core/Core.lua"]:122: in function <Interface/AddOns/AdiBags/core/Core.lua:82>
[string "=[C]"]: ?
[string "=[C]"]: ?
[string "@Interface/AddOns/AdiBags/libs/AceAddon-3.0/AceAddon-3.0.lua"]:66: in function <...ce/AddOns/AdiBags/libs/AceAddon-3.0/AceAddon-3.0.lua:61>
[string "@Interface/AddOns/AdiBags/libs/AceAddon-3.0/AceAddon-3.0.lua"]:494: in function `InitializeAddon'
[string "@Interface/AddOns/AdiBags/libs/AceAddon-3.0/AceAddon-3.0.lua"]:619: in function <...ce/AddOns/AdiBags/libs/AceAddon-3.0/AceAddon-3.0.lua:611>

Locals: <none>



Message: Interface/AddOns/AdiBags/core/Core.lua:122: attempt to call field 'SetCVarBool' (a nil value)
Time: Wed Jan  4 20:58:30 2023
Count: 1
Stack: Interface/AddOns/AdiBags/core/Core.lua:122: attempt to call field 'SetCVarBool' (a nil value)
[string "@Interface/AddOns/AdiBags/core/Core.lua"]:122: in function <Interface/AddOns/AdiBags/core/Core.lua:82>
[string "=[C]"]: ?
[string "=[C]"]: ?
[string "@Interface/AddOns/AdiBags/libs/AceAddon-3.0/AceAddon-3.0.lua"]:66: in function <...ce/AddOns/AdiBags/libs/AceAddon-3.0/AceAddon-3.0.lua:61>
[string "@Interface/AddOns/AdiBags/libs/AceAddon-3.0/AceAddon-3.0.lua"]:494: in function `InitializeAddon'
[string "@Interface/AddOns/AdiBags/libs/AceAddon-3.0/AceAddon-3.0.lua"]:619: in function <...ce/AddOns/AdiBags/libs/AceAddon-3.0/AceAddon-3.0.lua:611>

Locals: <none>

**Comment by Milky063** [comment]
3x AdiBags/core/Hooks.lua:216: attempt to call field 'CloseSpecialWindows' (a nil value)
[string "@AdiBags/core/Hooks.lua"]:216: in function <AdiBags/core/Hooks.lua:215>
[string "=(tail call)"]: ?
[string "=[C]"]: ?
[string "=[C]"]: in function `securecall'
[string "@FrameXML/UIParent.lua"]:3393: in function `CloseWindows'
[string "@FrameXML/UIParent.lua"]:3410: in function <FrameXML/UIParent.lua:3408>
[string "=[C]"]: in function `securecall'
[string "@FrameXML/UIParent.lua"]:4059: in function <FrameXML/UIParent.lua:4001>
[string "=[C]"]: in function `ToggleGameMenu'
[string "TOGGLEGAMEMENU"]:1: in function <[string "TOGGLEGAMEMENU"]:1>

Locals:
self = <table> {
 AcquireItemButton = <function> defined @AdiBags/widgets/ItemButton.lua:176
 modules = <table> {
 }
 UpdateFilters = <function> defined @AdiBags/core/Filters.lua:83
 SecureHook = <function> defined @AdiBags/libs/AceHook-3.0/AceHook-3.0.lua:335
 GetInteractingWindow = <function> defined @AdiBags/core/Core.lua:461
 GetContainerItemTexture = <function> defined @AdiBags/core/Utility.lua:438
 ConfigChanged = <function> defined @AdiBags/core/Core.lua:381
 GetSlotId = <function> defined @AdiBags/core/Utility.lua:70
 Filter = <function> defined @AdiBags/core/Filters.lua:144
 IsEnabled = <function> defined @AdiBags/libs/AceAddon-3.0/AceAddon-3.0.lua:447
 Reconfigure = <function> defined @AdiBags/core/Core.lua:211
 SetupTooltip = <function> defined @AdiBags/core/Utility.lua:127
 RegisterMessage = <function> defined @AdiBags/libs/CallbackHandler-1.0/CallbackHandler-1.0.lua:80
 UnregisterMessage = <function> defined @AdiBags/libs/CallbackHandler-1.0/CallbackHandler-1.0.lua:143
 hooks = <table> {
 }
 GetDistinctItemID = <function> defined @AdiBags/core/Utility.lua:303
 CreateFont = <function> defined @AdiBags/core/Fonts.lua:122
 isClassic = false
 db = <table> {
 }
 OpenOptions = <function> defined @AdiBags/core/Core.lua:296
 SetEnabledState = <function> defined @AdiBags/libs/AceAddon-3.0/AceAddon-3.0.lua:420
 Hook = <function> defined @AdiBags/libs/AceHook-3.0/AceHook-3.0.lua:273
 IterateBags = <function> defined @AdiBags/core/Bags.lua:190
 AcquireColumn = <function> defined @AdiBags/core/OO.lua:215
 OnEnable = <function> defined @AdiBags/core/Core.lua:128
 HookBagFrameCreation = <function> defined @AdiBags/core/Bags.lua:318
 NewBag = <function> defined @AdiBags/core/Bags.lua:157
 GetClass = <function> defined @AdiBags/core/OO.lua:129
 RegisterFilter = <function> defined @AdiBags/core/Filters.lua:120
 BACKDROP = <table> {
 }
 RemoveTooltip = <function> defined @AdiBags/core/Utility.lua:160
 UnregisterAllEvents = <function> defined @AdiBags/libs/CallbackHandler-1.0/CallbackHandler-1.0.lua:164
 isBCC = false
 GetContainerItemStackCount = <function> defined @AdiBags/core/Utility.lua:434
 IterateFilters = <function> defined @AdiBags/core/Filters.lua:116
 ReagentBankUpdated = <function> defined @AdiBags/core/Core.lua:372
 CreateBagSlotPanel = <function> defined @AdiBags/widgets/BagSlots.lua:448
 TOP_PADDING = 32
 hookedBags = <table> {
 }
 OpenBag = <function> defined @AdiBags/core/Hooks.lua:144
 AcquireDropzone = <function> defined @AdiBags/core/OO.lua:215
 ToggleAllBags = <function> defined @AdiBags/core/Hooks.lua:93
 OnProfileChanged = <function> defined @AdiBags/core/Core.lua:219
 GetName = <function> defined @AdiBags/libs/AceAddon-3.0/AceAddon-3.0.lua:275
 name = "AdiBags"
 OpenBackpack = <function> defined @AdiBags/core/Hooks.lua:175
 LayoutBags = <function> defined @AdiBags/core/Layout.lua:103
 Disable = <function> defined @AdiBags/libs/AceAddon-3.0/AceAddon-3.0.lua:310
 SetGlobalLock = <function> defined @AdiBags/core/Core.lua:429
 ParseItemLink = <function> defined @AdiBags/core/Utility.lua:178
 DisableHooks = <function> defined @AdiBags/core/Core.lua:198
 UpdateInteractingWindow = <function> defined @AdiBags/core/Core.lua:447
 Print = <function> defined @AdiBags/libs/AceConsole-3.0/AceConsole-3.0.lua:50
 OpenAllBags = <function> defined @AdiBags/core/Hooks.lua

**Comment by asymptotic1** [comment]
Update:  tried the "NoLib" version with a standard Ace lib installed and am getting the same errors.

**Comment by asymptotic1** [comment]
> Same. Easiest way to trigger it to hit escape.
> 
> ```
> Message: Interface/AddOns/AdiBags/core/Core.lua:122: attempt to call field 'SetCVarBool' (a nil value)
> Time: Wed Jan  4 20:58:30 2023
> Count: 1
> Stack: Interface/AddOns/AdiBags/core/Core.lua:122: attempt to call field 'SetCVarBool' (a nil value)
> [string "@Interface/AddOns/AdiBags/core/Core.lua"]:122: in function <Interface/AddOns/AdiBags/core/Core.lua:82>
> [string "=[C]"]: ?
> [string "=[C]"]: ?
> [string "@Interface/AddOns/AdiBags/libs/AceAddon-3.0/AceAddon-3.0.lua"]:66: in function <...ce/AddOns/AdiBags/libs/AceAddon-3.0/AceAddon-3.0.lua:61>
> [string "@Interface/AddOns/AdiBags/libs/AceAddon-3.0/AceAddon-3.0.lua"]:494: in function `InitializeAddon'
> [string "@Interface/AddOns/AdiBags/libs/AceAddon-3.0/AceAddon-3.0.lua"]:619: in function <...ce/AddOns/AdiBags/libs/AceAddon-3.0/AceAddon-3.0.lua:611>
> 
> Locals: <none>
> ```
> 
> ```
> Message: Interface/AddOns/AdiBags/core/Hooks.lua:216: attempt to call field 'CloseSpecialWindows' (a nil value)
> Time: Wed Jan  4 20:58:38 2023
> Count: 2
> Stack: Interface/AddOns/AdiBags/core/Hooks.lua:216: attempt to call field 'CloseSpecialWindows' (a nil value)
> [string "@Interface/AddOns/AdiBags/core/Hooks.lua"]:216: in function <Interface/AddOns/AdiBags/core/Hooks.lua:215>
> [string "=(tail call)"]: ?
> [string "=[C]"]: ?
> [string "=[C]"]: in function `securecall'
> [string "@Interface/FrameXML/UIParent.lua"]:3393: in function `CloseWindows'
> [string "@Interface/FrameXML/UIParent.lua"]:3410: in function <Interface/FrameXML/UIParent.lua:3408>
> [string "=[C]"]: in function `securecall'
> [string "@Interface/FrameXML/UIParent.lua"]:4059: in function `ToggleGameMenu'
> [string "TOGGLEGAMEMENU"]:1: in function <[string "TOGGLEGAMEMENU"]:1>
> 
> Locals: self = <table> {
>  AcquireItemButton = <function> defined @Interface/AddOns/AdiBags/widgets/ItemButton.lua:176
>  modules = <table> {
>  }
>  UpdateFilters = <function> defined @Interface/AddOns/AdiBags/core/Filters.lua:83
>  SecureHook = <function> defined @Interface/AddOns/AdiBags/libs/AceHook-3.0/AceHook-3.0.lua:335
>  GetInteractingWindow = <function> defined @Interface/AddOns/AdiBags/core/Core.lua:461
>  GetContainerItemTexture = <function> defined @Interface/AddOns/AdiBags/core/Utility.lua:438
>  ConfigChanged = <function> defined @Interface/AddOns/AdiBags/core/Core.lua:381
>  GetSlotId = <function> defined @Interface/AddOns/AdiBags/core/Utility.lua:70
>  Filter = <function> defined @Interface/AddOns/AdiBags/core/Filters.lua:144
>  IsEnabled = <function> defined @Interface/AddOns/AdiBags/libs/AceAddon-3.0/AceAddon-3.0.lua:447
>  Reconfigure = <function> defined @Interface/AddOns/AdiBags/core/Core.lua:211
>  SetupTooltip = <function> defined @Interface/AddOns/AdiBags/core/Utility.lua:127
>  RegisterMessage = <function> defined @Interface/AddOns/AdiBags/libs/CallbackHandler-1.0/CallbackHandler-1.0.lua:80
>  UnregisterMessage = <function> defined @Interface/AddOns/AdiBags/libs/CallbackHandler-1.0/CallbackHandler-1.0.lua:143
>  hooks = <table> {
>  }
>  GetDistinctItemID = <function> defined @Interface/AddOns/AdiBags/core/Utility.lua:303
>  CreateFont = <function> defined @Interface/AddOns/AdiBags/core/Fonts.lua:122
>  isClassic = false
>  db = <table> {
>  }
>  OpenOptions = <function> defined @Interface/AddOns/AdiBags/core/Core.lua:296
>  SetEnabledState = <function> defined @Interface/AddOns/AdiBags/libs/AceAddon-3.0/AceAddon-3.0.lua:420
>  Hook = <function> defined @Interface/AddOns/AdiBags/libs/AceHook-3.0/AceHook-3.0.lua:273
>  IterateBags = <function> defined @Interface/AddOns/AdiBags/core/Bags.lua:190
>  AcquireColumn = <function> defined @Interface/AddOns/AdiBags/core/OO.lua:215
>  OnEnable = <function> defined @Interface/AddOns/AdiBags/core/Core.lua:128
>  HookBagFrameCreation = <function> defined @Interface/AddOns/AdiBags/core/Bags.lua:318
>  NewBag = <function> defined @Interface/AddOns/AdiBags/core/Bags.lua:157
>  GetClass = <function> defined @Interface/AddOns/AdiBags/core/OO.lua:129
>  RegisterFilter = <function> defined @Interface/AddOns/AdiBags/core/Filters.lua:120
>  BACKDROP = <table> {
>  }
>  RemoveTooltip = <function> defined @Interface/AddOns/AdiBags/core/Utility.lua:160
>  UnregisterAllEvents = <function> defined @Interface/AddOns/AdiBags/libs/CallbackHandler-1.0/CallbackHandler-1.0.lua:164
>  isBCC = false
>  GetContainerItemStackCount = <function> defined @Interface/AddOns/AdiBags/core/Utility.lua:434
>  IterateFilters = <function> defined @Interface/AddOns/AdiBags/core/Filters.lua:116
>  ReagentBankUpdated = <function> defined @Interface/AddOns/AdiBags/core/Core.lua:372
>  CreateBagSlotPanel = <function> defined @Interface/AddOns/AdiBags/widgets/BagSlots.lua:448
>  TOP_PADDING = 32
>  hookedBags = <table> {
>  }
>  OpenBag = <function> defined @Interface/AddOns/AdiBags/core/Hooks.lua:144
>  AcquireDropzone = <function> defined @Interface/AddOns/AdiBags/core/OO.lua:215
>  ToggleAllBags = <function> defined @Interface/AddOns/AdiBags/core/Hooks.lua:93
>  OnProfileCha
> ```
> 
> ```
> Message: Interface/AddOns/AdiBags/core/Hooks.lua:216: attempt to call field 'CloseSpecialWindows' (a nil value)
> Time: Wed Jan  4 20:58:38 2023
> Count: 2
> Stack: Interface/AddOns/AdiBags/core/Hooks.lua:216: attempt to call field 'CloseSpecialWindows' (a nil value)
> [string "@Interface/AddOns/AdiBags/core/Hooks.lua"]:216: in function <Interface/AddOns/AdiBags/core/Hooks.lua:215>
> [string "=(tail call)"]: ?
> [string "=[C]"]: ?
> [string "=[C]"]: in function `securecall'
> [string "@Interface/FrameXML/UIParent.lua"]:3393: in function <Interface/FrameXML/UIParent.lua:3362>
> [string "=[C]"]: in function `securecall'
> [string "@Interface/FrameXML/UIParent.lua"]:2689: in function `ShowUIPanel'
> [string "@Interface/FrameXML/UIParent.lua"]:2598: in function <Interface/FrameXML/UIParent.lua:2594>
> [string "=[C]"]: in function `SetAttribute'
> [string "@Interface/FrameXML/UIParent.lua"]:3196: in function `ShowUIPanel'
> [string "@Interface/FrameXML/UIParent.lua"]:4078: in function `ToggleGameMenu'
> [string "TOGGLEGAMEMENU"]:1: in function <[string "TOGGLEGAMEMENU"]:1>
> 
> Locals: self = <table> {
>  AcquireItemButton = <function> defined @Interface/AddOns/AdiBags/widgets/ItemButton.lua:176
>  modules = <table> {
>  }
>  UpdateFilters = <function> defined @Interface/AddOns/AdiBags/core/Filters.lua:83
>  SecureHook = <function> defined @Interface/AddOns/AdiBags/libs/AceHook-3.0/AceHook-3.0.lua:335
>  GetInteractingWindow = <function> defined @Interface/AddOns/AdiBags/core/Core.lua:461
>  GetContainerItemTexture = <function> defined @Interface/AddOns/AdiBags/core/Utility.lua:438
>  ConfigChanged = <function> defined @Interface/AddOns/AdiBags/core/Core.lua:381
>  GetSlotId = <function> defined @Interface/AddOns/AdiBags/core/Utility.lua:70
>  Filter = <function> defined @Interface/AddOns/AdiBags/core/Filters.lua:144
>  IsEnabled = <function> defined @Interface/AddOns/AdiBags/libs/AceAddon-3.0/AceAddon-3.0.lua:447
>  Reconfigure = <function> defined @Interface/AddOns/AdiBags/core/Core.lua:211
>  SetupTooltip = <function> defined @Interface/AddOns/AdiBags/core/Utility.lua:127
>  RegisterMessage = <function> defined @Interface/AddOns/AdiBags/libs/CallbackHandler-1.0/CallbackHandler-1.0.lua:80
>  UnregisterMessage = <function> defined @Interface/AddOns/AdiBags/libs/CallbackHandler-1.0/CallbackHandler-1.0.lua:143
>  hooks = <table> {
>  }
>  GetDistinctItemID = <function> defined @Interface/AddOns/AdiBags/core/Utility.lua:303
>  CreateFont = <function> defined @Interface/AddOns/AdiBags/core/Fonts.lua:122
>  isClassic = false
>  db = <table> {
>  }
>  OpenOptions = <function> defined @Interface/AddOns/AdiBags/core/Core.lua:296
>  SetEnabledState = <function> defined @Interface/AddOns/AdiBags/libs/AceAddon-3.0/AceAddon-3.0.lua:420
>  Hook = <function> defined @Interface/AddOns/AdiBags/libs/AceHook-3.0/AceHook-3.0.lua:273
>  IterateBags = <function> defined @Interface/AddOns/AdiBags/core/Bags.lua:190
>  AcquireColumn = <function> defined @Interface/AddOns/AdiBags/core/OO.lua:215
>  OnEnable = <function> defined @Interface/AddOns/AdiBags/core/Core.lua:128
>  HookBagFrameCreation = <function> defined @Interface/AddOns/AdiBags/core/Bags.lua:318
>  NewBag = <function> defined @Interface/AddOns/AdiBags/core/Bags.lua:157
>  GetClass = <function> defined @Interface/AddOns/AdiBags/core/OO.lua:129
>  RegisterFilter = <function> defined @Interface/AddOns/AdiBags/core/Filters.lua:120
>  BACKDROP = <table> {
>  }
>  RemoveTooltip = <function> defined @Interface/AddOns/AdiBags/core/Utility.lua:160
>  UnregisterAllEvents = <function> defined @Interface/AddOns/AdiBags/libs/CallbackHandler-1.0/CallbackHandler-1.0.lua:164
>  isBCC = false
>  GetContainerItemStackCount = <function> defined @Interface/AddOns/AdiBags/core/Utility.lua:434
>  IterateFilters = <function> defined @Interface/AddOns/AdiBags/core/Filters.lua:116
>  ReagentBankUpdated = <function> defined @Interface/AddOns/AdiBags/core/Core.lua:372
>  CreateBagSlotPanel = <function> defined @Interface/AddOns/AdiBags/widgets/BagSlots.lua:448
>  TOP_PADDING = 32
>  hookedBags = <table> {
>  }
>  OpenBag = <function> define
> ```

Update: tried the "NoLib" version with a standard Ace lib installed and am getting the same errors.

**Comment by amonr3** [comment]
5x AdiBags/core/Core.lua:122: attempt to call field 'SetCVarBool' (a nil value)
[string "@AdiBags/core/Core.lua"]:122: in function <AdiBags/core/Core.lua:82>
[string "=[C]"]: ?
[string "=[C]"]: ?
[string "@AdiBags/libs/AceAddon-3.0-13/AceAddon-3.0.lua"]:66: in function <...ceAdiBags/libs/AceAddon-3.0/AceAddon-3.0.lua:61>
[string "@AdiBags/libs/AceAddon-3.0-13/AceAddon-3.0.lua"]:494: in function `InitializeAddon'
[string "@AdiBags/libs/AceAddon-3.0-13/AceAddon-3.0.lua"]:619: in function <...ceAdiBags/libs/AceAddon-3.0/AceAddon-3.0.lua:611>

yep same error.

we just had to downgrade to verison 
AdiBags-v1.9.50.zip on 11/30/2022
![image](https://user-images.githubusercontent.com/121996481/210699616-2f6a9b0d-5619-4340-923e-c34b9ff19976.png)

**Comment by Cidan** [comment]
Fixed with 1.9.52, which is going out in a few minutes.
